app/models/SearchHits.ts

```diff
@@ -4,20 +4,28 @@ import { ScheduleDay, parseAlgoliaSchedule } from "./Schedule";
 import { PhoneNumber } from "./Meta";
 import { RecurringSchedule } from "./RecurringSchedule";
 
+interface BaseHit {
+  _geoloc: { lat: number; lng: number };
+  is_mohcd_funded: boolean;
+  resource_id: number;
+}
+
 export interface ServiceHit
-  extends Omit<Service, "schedule" | "recurringSchedule" | "instructions"> {
+  extends Omit<Service, "schedule" | "recurringSchedule" | "instructions">,
+    BaseHit {
   type: "service";
   instructions: string[];
   phones: PhoneNumber[];
   recurringSchedule: RecurringSchedule | null;
-  resource_id: number;
```

Comment:
I think ServiceHits do have a `resource_id` prop on them, which is helpful for looking up the parent resource. But maybe I'm misunderstanding the purpose...

Reply:
Ah, for this, I moved it into the `BaseHit` interface I defined above, since I wanted to share a number of fields between the `ServiceHit`s and `OrganizationHit`s. I think I was getting a type error somewhere because `OrganizationHit` actually has a `resource_id` as well, and we were expecting it to have it in some code somewhere, and instead of maintaining two separate lists, I thought to create a shared type between `ServiceHits` and `OrganizationHit`s.

Let me know if this sounds OK. I can also remove the shared interface and separately list out the common fields that we depend on in each type.